tests/secret-providers/aws_secretmanager_pod_identity/aws_secretmanager_pod_identity_test.go

```diff
@@ -141,6 +142,31 @@ spec:
       name: {{.SecretManagerSecretName}}
 `
 
+	triggerAuthenticationSecretKeyTemplate = `apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: {{.TriggerAuthenticationName}}
+  namespace: {{.TestNamespace}}
+spec:
+  awsSecretManager:
+    credentials:
+      accessKey:
+        valueFrom:
+          secretKeyRef:
+            name: {{.AwsCredentialsSecretName}}
+            key: AWS_ACCESS_KEY_ID
+      accessSecretKey:
+        valueFrom:
+          secretKeyRef:
+            name: {{.AwsCredentialsSecretName}}
+            key: AWS_SECRET_ACCESS_KEY
+    region: {{.AwsRegion}}
+    secrets:
+    - parameter: connection
+      name: {{.SecretManagerSecretName}}
+	  secretKey: connectionString
```

Comment:
This indentation does not show up on my machine. Not sure why it is here

Reply:
it produces e2e failures :(  Could you fix it? Maybe you need to make 2 commits as usually tabs and whitespace are ignored depending on config